ArduPlane/Parameters.cpp

```diff
@@ -664,6 +669,8 @@ const AP_Param::Info Plane::var_info[] = {
     // @DisplayName: Pitch angle offset
     // @Description: Offset applied to AHRS pitch used for in-flight pitch trimming. Correct ground leveling is better than changing this parameter.
     // @Units: cdeg
+    // @Range: -4500 4500
+    // @Increment: 1
```

Comment:
maybe 10 (.1 deg)? otherwise it will take forever to change in your scripts

Reply:
we're using incrementation of 1 on all cdeg unit params (like LIM_ROLL_CD etc) so why diverge here? is this causing any issues on other params with the same unit / incrementation? in that case rather change the others too, that would likley exceed the scope of this PR though.

Reply:
good devcall question....cant see why any Centidegree param really needs .01 deg precision....makes incrementers take forever in GCS...will begin up....want to sleep and let me bring up?

Reply:
@Hwurzburg would highly appreciate that, thanks henry! btw param osd does a pretty good job guessing increments with the option to set individually, so i personally have never actually run into increment setting issues.

Reply:
definitely use 10

Reply:
done